src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -283,3 +299,37 @@ def find_version_info_in_manifest_file(
             patch_name = version_info[patch_field]
 
     return version_name, patch_name
+
+
+def project_path_to_deploy_path(
```

Comment:
nit: this is singular, but acts on a list.

That being said, I do think the list input isn't particularly needed. You could use list comprehension on the caller side and keep this utility function singular for simplicity, something like:

`deploy_paths = [project_path_to_deploy_path(p) for p in project_paths]`